from ..error import PythImageError as PythImageError

